dbschema/progress-summary.esdl

```diff
@@ -0,0 +1,23 @@
+module default {
+  type ProgressSummary {
```

Comment:
I can't help but feel that this shouldn't be in the default module.
Maybe
```edgeql
ProgressReport::ProgressSummary
# or
ProgressReport::ProductProgress::Summary
```
Thoughts @andrewmurraydavid @bryanjnelson?

Reply:
Do we have a `ProductProgress` now?  

Ah, yes, Brent has a PR up for it.  I believe I like `ProgressReport::ProductProgress::Summary` then.

Reply:
Should it move to a separate file or can it stay here, just changing the namespace?

Reply:
Hmm...I could see moving it [over here](https://github.com/SeedCompany/cord-api-v3/pull/3091/files) maybe, after this is solidified?  I'll let @CarsonF weigh in.

Reply:
That's what I was thinking too, @bryanjnelson . That's why I suggested to wait until #3091 is merged.

Reply:
Ah...must've missed your edit!